### ISSUE TITLE
Export PDF - Text stroke goes wild

### ISSUE THREAD

**Issue by TheRadicalDreamer** [photopea/photopea#4715]
![image](https://user-images.githubusercontent.com/11801254/172758940-8c33272e-b45d-467f-a5bb-10198ee35b4a.png)

This issue again.

Why it shows differently when exporting?

**Comment by photopea** [comment]
Could you share a small PSD with me?

**Comment by TheRadicalDreamer** [comment]
Here: https://www.photopea.com/#iJ60jMF08

**Comment by TheRadicalDreamer** [comment]
@photopea did you discover something? 🥺

**Comment by TheRadicalDreamer** [comment]
it also happens when you export to SVG

**Comment by photopea** [comment]
All this is caused by the fact, that PDF and SVG do not support boolean operations on shapes (subtract, intersect, ... ) and they do not support outer stroke (they only have a center stroke).

Photopea merges all shapes in the layer when exporting a PDF. Merge them already in Photopea to see how it will look like (at the bottom under Union, Subtract, Intersect ...).

You use Miter joins. There is an important parameter "Miter-Limit": https://www.youtube.com/watch?v=vT-WLdYiLn8 . You can change this parameter now. I believe all your problems can be fixed by decreasing the miter limit.

**Comment by TheRadicalDreamer** [comment]
Good!! Thanks!

The only bad point, is that i have to export it a couple of times to see the exact limt i need.

But thank you!

**Comment by TheRadicalDreamer** [comment]
And here:
![image](https://user-images.githubusercontent.com/11801254/174643689-6c3b147a-028c-4163-97a4-cc824abb3222.png)


It wont exist??

**Comment by photopea** [comment]
Yes, the Stroke Layer Style around vector shapes is converted into a vector stroke, that is round (miter limit does not apply to round joins).

**Comment by TheRadicalDreamer** [comment]
I get it.

I was wondering if you are going to add it there or not.
